src/massive/munit/TestRunner.hx

```diff
@@ -189,7 +189,13 @@ class TestRunner implements IAsyncDelegateObserver
 
         #if (neko || cpp || java || hl || eval)
 		var self = this;
-		var runThread:Thread = Thread.create(function()
+		var createThread = Thread.create;
+        #if haxe4 
+        #if (haxe >= version("4.2.0-rc.1"))
```

Comment:
`#if (haxe> = version (" 4.2.0-rc.1 "))` doesn't work without `#if haxe4`?

Reply:
it works fine on Haxe 4, but not on Haxe 3

Reply:
ok ok, `haxe >= version("4.2.0-rc.1")` - already says that we are working with haxe 4, I don't understand why to specify again `# if haxe4`

Reply:
I didn't think it wouldn't compile, but I would still do one assignment instead of two))